bedrock/mozorg/templates/mozorg/about/forums/forums.html

```diff
@@ -36,16 +36,15 @@ <h1><span class="highlight">{{ self.page_title() }}</span></h1>
   {{ _('Mozilla newsgroups are sponsored by <a href="%s">Giganews Newsgroups</a>, to whom we are very grateful.')|format('https://www.giganews.com/') }}
   <a href="https://www.giganews.com/"><img src="{{ static('img/mozorg/about/giganews.png') }}" alt="" height="25" width="89" /></a>
   </p>
-
+  <a href="https://www.google.com/" ><img
```

Comment:
NIt: also suggest putting this `<img>` element on a single line vs splitting over multiple (this is a really old template)